pyproject.toml

```diff
@@ -49,6 +49,12 @@
 [tool.black]
 line-length = 100
 
+[tool.isort]
+force_single_line = true
+honor_noqa = true
```

Comment:
Could you try dropping this? Originally, I've added it to my repos when fighting some isort version upgrade compatibility issues but I don't remember why anymore. So if you find out why it's needed, maybe add a code comment about that too.